src/openrct2/world/Entrance.cpp

```diff
@@ -548,26 +548,26 @@ void maze_entrance_hedge_replacement(int32_t x, int32_t y, rct_tile_element* til
     x += CoordsDirectionDelta[direction].x;
     y += CoordsDirectionDelta[direction].y;
     int32_t z = tileElement->base_height;
-    int32_t rideIndex = track_element_get_ride_index(tileElement);
+    int32_t rideIndex = tileElement->properties.entrance.ride_index;
```

Comment:
Should this not use the `GetRideIndex()` function?

Reply:
The Entrance element does not yet have this function, AFAIK.

Reply:
To get the entrance index, you should shift it to the right by 4 spaces and AND with 3. `track_element_get_ride_index` didn't do any shifting or other bit operations, so this looks a bit odd to me.

I highly doubt this is meant to read from the entrance part, because a few lines later it breaks out of the loop when `tileElement->GetType() != TILE_ELEMENT_TYPE_TRACK`.

Reply:
I have no idea what you're talking about. We're talking about an entrance element here, which does not appear to save the station index in the `ride_index` byte (7).

Reply:
My bad, I thought you meant the entrance index, not ride index. There's no bit operations needed there.